UnitThree/view/frontend/templates/unitThreeTab.phtml

```diff
@@ -0,0 +1,17 @@
+<strong>Username::</strong>
+<?php echo $this->helper('RLTSquare\UnitThree\Helper\Data\Help')->getConfig('rltsquare2/general/username'); ?><br>
+<strong>Password::</strong>
+<?php
+$type = $this->helper('RLTSquare\UnitThree\Helper\Data\Help')->getConfig('rltsquare2/general/dropdown');
+$pass = $this->helper('RLTSquare\UnitThree\Helper\Data\Help')->getConfig('rltsquare2/general/password');
+if ($type == 'Development' || $type == 'Stagging') {
```

Comment:
Not good ... instead of 'Development' and 'Stagging', you could have declared constant in your options class, You should declare value keys here as constants, e.g. "const ENV_STAGING = 'staging'", so you can later use RLTSquare\UnitThree\Model\Config\Source\Select::ENV_STAGING in if statements and other comparisons you need.